src/display/editor/editor.js

```diff
@@ -599,6 +600,15 @@ class AnnotationEditor {
       this.parent.setActiveEditor(null);
     }
   }
+
+  /**
+   * Check if the editor has been changed.
+   * @param {Object} serialized
+   * @returns {boolean}
+   */
+  hasElementChanged(serialized = null) {
```

Comment:
There's nothing passing an argument to this method, so you could probably just change it to a getter instead?

Reply:
It'll be used later (in an other patch).